client/src/menu/Anchor.tsx

```diff
@@ -104,13 +105,21 @@ export default function Anchor(props: Readonly<{
     const setFontSize = (n: number) => {
         document.documentElement.style.fontSize = `${n}em`;
     }
+    const setAccessibilityFontEnabled = (enabled: boolean) => {
+        const font = enabled ? 'game-accessible' : 'game-base';
+        const iconFactor = enabled ? '1.2' : '1';
+
+        document.documentElement.style.setProperty('--game-font', font);
+        document.documentElement.style.setProperty('--icon-factor', iconFactor);
+    }

```

Comment:
You should also override the kira font

Reply:
How would I go about doing this? Is there a kira CSS file that has that stuff?

Reply:
This should be resolved now!